src/client/Microsoft.Identity.Client/TokenCacheNotificationArgs.cs

```diff
@@ -146,6 +147,44 @@ public sealed partial class TokenCacheNotificationArgs
             PiiLoggingEnabled = piiLoggingEnabled;
         }
 
+        /// <summary>
+        /// This constructor is for test purposes only. It allows apps to unit test their MSAL token cache implementation code.
+        /// </summary>
+        public TokenCacheNotificationArgs(    // only use this constructor in product code
```

Comment:
I'm confused on the comments (but same for other methods here so maybe a style thing? The method header says it's test purposes only and the other comment says it's only used for product code. That conflicts, no?